content/docs/command-reference/stage/add.md

```diff
@@ -0,0 +1,432 @@
+# stage add
+
+Helper command to add or update <abbr>stages</abbr> in `dvc.yaml`. Requires a
+name and a command.
+
+## Synopsis
+
+```usage
+usage: dvc stage add [-h] [-q | -v] -n <name> [-d <path>] [-o <path>]
+                     [-O <path>] [-p [<path>:]<params_list>] [-m <path>]
+                     [-M <path>] [--plots <path>] [--plots-no-cache <path>]
+                     [-w <path>] [-f]
+                     [--outs-persist <path>] [--outs-persist-no-cache <path>]
+                     [--always-changed] [--external] [--desc <text>]
+                    command
+
+positional arguments:
+  command               Command to execute.
+```
+
+## Description
+
+`dvc stage add` is a helper for creating or updating
+[pipeline](/doc/command-reference/dag) stages in a `dvc.yaml` file (located in
+the current working directory).
```

Comment:
```suggestion
Creates or updates stages in a [pipeline](/doc/command-reference/dag) (saved to
`dvc.yaml` in the current working directory).
```

Reply:
But how do you update vs. create?

Reply:
`update` just means overwrite here (with `--force`).

Reply:
Sure. My question was meant as a hint on what to improve in the explanation.

Reply:
Which at this point (since this is merge) I'll do separately, so np.